packages/loader/src/ktx2/KTX2Loader.ts

```diff
@@ -139,6 +139,9 @@ export class KTX2Loader extends Loader<Texture2D | TextureCube> {
   ): KTX2TargetFormat | null {
     for (let i = 0; i < priorityFormats.length; i++) {
       const capabilities = this._supportedMap[priorityFormats[i]];
+
+      if (!capabilities) return null;
```

Comment:
what kind of case can make `capabilities` null or undefined?

Reply:
the priorityFormatsValue is [0,4,1,2,3,7]

Reply:
it looks like  R8G8B8A8 is not supported